compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5065,17 +5066,6 @@ genericLoadHelper(TR::Node * node, TR::CodeGenerator * cg, TR::MemoryReference *
          }
       else //if (form == MemReg)
          {
-         if (cg->isConcurrentScavengeEnabled())
-            {
-            // TODO (GuardedStorage): If we are in the evaluation of a compressedrefs sequence and are about to generate
-            // compressed load we override the instruction opcode to generate a guarded load and shift instead. We should
-            // figure out a better way to handle this.
-            if (cg->isEvaluatingCompressionSequence() && load == TR::InstOpCode::LLGF)
-               {
-               load = TR::InstOpCode::LLGFSG;
-               }
-            }
-
          generateRXInstruction(cg, load, node, targetRegister, tempMR);
```

Comment:
Actually now looking at it this code was incorrect in its previous state. LLGFSG is an RXY instruction, not an RX instruction. Though I'm almost certain for all intents and purposes we would have generated the correct thing anyway.